views/package-lock.json

```diff
@@ -10,9 +10,8 @@
       "integrity": "sha512-Z/fRV8TSxitZwO40zeL8T+pLbxoqp5L2dDqWf+0kPOV3Guf2juP1eXZygiXhwmHHOTgwpSeKSC0sOJJATsL5xQ=="
     },
     "@oat-sa/tao-item-runner-qti": {
-      "version": "0.4.19",
-      "resolved": "https://registry.npmjs.org/@oat-sa/tao-item-runner-qti/-/tao-item-runner-qti-0.4.19.tgz",
-      "integrity": "sha512-scjmtp1LEZyyhzwz5LKjZ/0BO47cC3KjWrzoZGfaAKZW3bn8zJpbTCI5aDQDOD8A3FPBfHY0IFs87jsHyG4CWA=="
+      "version": "github:oat-sa/tao-item-runner-qti-fe#ca1ac75dfe0377ab9b922b52bcbdf63d524ca3be",
```

Comment:
Not right, should be npm version
`rm -rf node_modules/`
`npm install`

Reply:
`npm update` I believe, `install` will use the lock file's data.